src/posit/connect/oauth/associations.py

```diff
@@ -66,9 +70,11 @@ def delete(self) -> None:
         path = f"v1/content/{self.content_guid}/oauth/integrations/associations"
         self._ctx.client.put(path, json=data)
 
-    def update(self, integration_guid: str) -> None:
+    def update(self, integration_guid: str | list[str]) -> None:
```

Comment:
In these situations, I've found that its better to do one or the other (str or list[str]).